src/components/MessageBox/index.tsx

```diff
@@ -18,6 +25,22 @@ type SubTabType = {
   hoverClass: string;
 };
 
+const getTabs = (
+  ownerRequest?: OwnerRequestType[],
+  userRequests?: UserRequestType[],
+): Tab[] => {
+  const tabs: (Tab | null)[] = [
+    ownerRequest?.length
+      ? { value: 'received', label: 'Příchozí žádosti', items: ownerRequest }
+      : null,
+    userRequests?.length
```

Comment:
You  could also do a local mutation.

if(ownerRequest?.length) {
tabs.push({ value: 'received', label: 'Příchozí žádosti', items: ownerRequest }) 
}

. 
.
return tabs